model/registry/artifact.yaml

```diff
@@ -0,0 +1,60 @@
+groups:
+  - id: registry.artifact
+    prefix: artifact
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to artifacts.
```

Comment:
Have added a definition that is consistent with SLSA's definition at the attribute group layer.

Reply:
so it seems a maven/nuget/npm/etc  package is an artifact. In this case, the file name is not the main characteristics of it. And I'd be surprised to see a file name in `artifact.name`, I'd expected a package name (public, used for distribution).

We have purl as a package identifier, so we don't necessarily need alternative package name, so maybe we can rename `artifact.name` to something like `artifact.build.name | filename` so that's more specific and less surprising?

Reply:
There's slight minutia on that front. At least according to the [SLSA documentation](https://slsa.dev/spec/v1.0/terminology#mapping-to-real-world-ecosystems), Maven maps to an `artifact` while NPM maps to a `package`. 

Regardless of that slight adjustment, I think you're right. Opting to convert this to `artifact.filename` to properly map to the `attestation.filename` value & stay consistent to the SLSA documentation.

Reply:
I've adjusted the documentation accordingly.